bot/controllers/games.js

```diff
@@ -10,11 +11,77 @@ class GamesController extends BaseController {
     this.commands = [
       new Command(
         '!8ball',
+        '!8ball <question>',
+        'Shakes 8 Ball',
+        '',
```

Comment:
Be sure to add a description to this command.

Reply:
Thank you for this I overlooked this one fixing now